api/src/test/java/com/epam/pipeline/manager/pipeline/RunScheduleManagerTest.java

```diff
@@ -103,214 +117,238 @@ public void setUp() {
         runScheduleVO.setTimeZone(TIME_ZONE.getDisplayName());
         runScheduleVO.setCronExpression(cronExpression);
         runScheduleVO.setAction(action);
         return runScheduleVO;
     }
 
-    private void createPipelineRun(final Long runId, final Long pipelineId) {
+    private PipelineRun createPipelineRun(final Long runId, final Long pipelineId) {
         final AbstractCloudRegion cloudRegion = ObjectCreatorUtils.getDefaultAwsRegion();
-        regionDao.create(cloudRegion);
         PipelineRun run = ObjectCreatorUtils.createPipelineRun(runId, pipelineId, null, cloudRegion.getId());
         run.setStatus(TaskStatus.RUNNING);
         run.getInstance().setSpot(false);
-        pipelineRunDao.createPipelineRun(run);
+        return run;
     }
 
     private RunConfiguration createRunConfiguration() {
-        //create
-        RunConfigurationEntry entry =
-                ObjectCreatorUtils.createConfigEntry("entry", true, null);
+        final RunConfigurationEntry entry =
+                ObjectCreatorUtils.createConfigEntry(ENTRY_NAME, true, null);
 
-        RunConfiguration configuration =
+        final RunConfiguration configuration =
                 ObjectCreatorUtils.createConfiguration(TEST_NAME, null, null,
                         USER_OWNER, Collections.singletonList(entry));
-        return configurationDao.create(configuration);
+        return configuration;
     }
 
+    private RunSchedule createRunSchedule() {
+        final RunSchedule runSchedule = new RunSchedule();
+        runSchedule.setUser(USER_OWNER);
+        runSchedule.setAction(RunScheduledAction.PAUSE);
+        runSchedule.setSchedulableId(RUN_ID);
+        runSchedule.setType(ScheduleType.PIPELINE_RUN);
+        runSchedule.setCronExpression(CRON_EXPRESSION1);
+        runSchedule.setCreatedDate(new Date());
+        runSchedule.setTimeZone(TimeZone.getTimeZone("GMT"));
+        return runSchedule;
+    }
 
-    private void loadAndAssertSchedule(final RunSchedule schedule) {
-        final RunSchedule loadRunSchedule = runScheduleManager.loadSchedule(schedule.getId());
-        assertEquals(schedule.getSchedulableId(), loadRunSchedule.getSchedulableId());
-        assertEquals(schedule.getType(), loadRunSchedule.getType());
-        assertEquals(schedule.getCronExpression(), loadRunSchedule.getCronExpression());
-        assertEquals(schedule.getAction(), loadRunSchedule.getAction());
-        assertEquals(schedule.getUser(), loadRunSchedule.getUser());
+    private RunSchedule createRunSchedule(final Long id, final Long schedulableId,
+                                          final RunScheduledAction action, final String cronExpression) {
+        final RunSchedule runSchedule = new RunSchedule();
+        runSchedule.setUser(USER_OWNER);
+        runSchedule.setAction(action);
+        runSchedule.setId(id);
+        runSchedule.setSchedulableId(schedulableId);
```

Comment:
You can inline `RUN_ID` here removing extra method parameter `schedulableId`.

```
runSchedule.setSchedulableId(RUN_ID);
```

Reply:
Do you use Intellij IDEA? It seems like such an issue should be highlighted.
![image](https://user-images.githubusercontent.com/16464586/109777615-fa921580-7c14-11eb-8869-317579c853a7.png)